packages/medusa/src/types/money-amount.ts

```diff
@@ -0,0 +1,5 @@
+export enum MoneyAmountType {
```

Comment:
Is `MoneyAmountType` still relevant now we got `PriceListType`, it doesn't seem to be used.

I'm thinking a default money amount is a money amount where `price_list_id = null`.